static/scss/detail/text-section.scss

```diff
@@ -0,0 +1,52 @@
+// sass-lint:disable mixins-before-declarations
+.text-block {
+  width: 100%;
+  padding: 100px 0 50px;
+  background: white url("#{$static-path}/images/transparent-pixels-bg.png") repeat-x 0 100%;
+  min-height: 400px;
+  overflow: hidden;
+  position: relative;
+  color: black;
+  font-size: 20px;
+  line-height: 28px;
+  font-weight: 500;
+
+  [class^="col"],
+  [class*=" col"],
+  .container {
+    @include media-breakpoint-up(lg) {
+      position: static;
+      min-height: 505px;
+    }
+  }
+
+  .action-button {
+    font-size: x-large;
+  }
+
+  h1 {
+    font-size: 40px;
+    line-height: 45px;
+    color: $primary;
+    font-weight: 600;
+    margin: 0 0 10px;
+    text-transform: uppercase;
```

Comment:
instead of this, we can use bootstrap class

Reply:
Removed, it was redundant anyway. I'd already added the bootstrap class in the template, forgot to remove this from the styles.